src/Compilers/Core/Portable/Emit/EditAndContinue/SymbolMatcher.cs

```diff
@@ -131,7 +131,7 @@ internal abstract class SymbolMatcher
             return result;
         }
 
-        private IReadOnlyDictionary<string, AnonymousTypeValue> MapAnonymousDelegatesWithFixedTypes(IReadOnlyDictionary<string, AnonymousTypeValue> anonymousDelegates)
+        private IReadOnlyDictionary<string, AnonymousTypeValue> MapAnonymousDelegatesWithIndexedNames(IReadOnlyDictionary<string, AnonymousTypeValue> anonymousDelegates)
```

Comment:
>MapAnonymousDelegatesWithIndexedNames [](http://example.com/codeflow?start=64&length=37)

Do we need to make any adjustments beyond the rename? Perhaps, taking default values and params into account in new scenarios, like you've done in another PR?